### PR TITLE
Added feature to allow test file as spec.js & Extended it to create t…

### PR DESCRIPTION
**Pull request by chandanrjit**
Hi , it is wonderful module and  i have add few changes  which may required by others 

- Added to create the Storybook component (Conditional based)
- Allowed options to change "test" file as "spec.js"  .

### CODE REVIEW

**Comment by snaerth** [comment]
@chandanrjit Great addition! I will have a look at it soon

File: lib/index.js [hunk 1]
```diff
@@ -84,9 +86,15 @@ function createFiles(componentName, componentPath, cssFileExt) {
     const prettierParser = isTypeScript ? 'typescript' : 'babylon';
 
     if (!notest && !(config && config.includes('notest'))) {
-      files.push(`${name}.test.${ext}`);
+      if((config && config.includes('spec')))
```
Comment (snaerth) [inline lib/index.js]: I use curly braces in my projects :)

File: lib/utils/stringHelper.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
-
+/**
+ *Capatalize every first letter in component name
+ *button-click --> Button-Click for Stroy book
+ * @param {*} string
+ * @returns
+ */
+function componentNameWithoutSpecialCharacter(string){
```
Comment (snaerth) [inline lib/utils/stringHelper.js]: This function can be much shorter and you should remove lodash. Please write it like this

```
/**
 * Removes special characters from string makes every camel case character uppercase
 * 
 * @param {String} str - String to transform
 * @returns {String}
 */
function componentNameWithoutSpecialCharacter(str){
  return str.split('-').map(capitalizeFirstLetter).join('')
}
```

**Comment by snaerth** [comment]
@chandanrjit

**Comment by chandanrjit** [comment]
updated PR with recommedations

**Pull request merged**